test/BUILD

```diff
@@ -328,3 +328,11 @@ scala_specs2_junit_test(
     data = ["//src/java/com/google/devtools/build/lib:worker"],
     jvm_flags = ["-Dlocation.expanded=$(location //src/java/com/google/devtools/build/lib:worker)"],
 )
+
+scala_junit_test(
+    name = "JunitFiltersAbstractClassesAndInterfaces",
+    srcs = ["src/main/scala/scala/test/junit/JunitAbstractClassAndInterface.scala"],
+    size = "small",
+    suffixes = ["Test"],
+    print_discovered_classes = True
+)
```

Comment:
The test indeed failed before my change. It failed since JUnit (and specs2) fail when trying to instantiate traits and abstract classes via reflection.
To me targets in this package are somewhere between a unit test and a full blown e2e (which resides in `run_test.sh`).
If I can add a test here and not in `run_test.sh` I'd rather do it since the feedback cycle is much faster.